### ISSUE TITLE
ie issues

### ISSUE THREAD

**Issue by astratagem** [templesmc/tutv#31]
also see #10 flexslider nav gone or inaccessible
#### IE8
- clearfix hack appears broken.
- grid is broken (sometimes)
- **cannot scroll!**
- gradients are fucked. use backup image?
#### IE9
- gradients are fucked. use backup image?

**Comment by astratagem** [comment]
scrolling fixed in 34dc3dcc05dd3cd960dd3ea83eb88479e3266f60 with removal of sticky footer

**Comment by astratagem** [comment]
the broken grid issue is probably caused by block-level elements contained within `<a>` elements. either change this markup, or take steps towards using the html5 doctype (including modernizr etc)

**Comment by astratagem** [comment]
blocks and other layout issues fixed by specifying the ie document mode:

```
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
```

http://alistapart.com/article/beyonddoctype
http://msdn.microsoft.com/en-us/library/cc288325(v=vs.85).aspx

this would be required for moving towards html5

**Comment by astratagem** [comment]
this will all be resolved, i believe, by fixing #28 and including the line of code mentioned in the above comment.

**Comment by astratagem** [comment]
alas, such is not the case...

**Comment by bpizzle** [comment]
so sad D':

**Comment by astratagem** [comment]
@bpizzle i think the layout is broke because i'm using the `rem` unit in some places. [not the best idea](http://caniuse.com/#feat=rem).

will test tomorrow.

**Comment by astratagem** [comment]
removing rem unit helped out with vertical spacing, but horizontally, ie8 still seems to be ignoring `alpha` and `omega` classes.

[opened question on stackoverflow](http://stackoverflow.com/questions/16309437/960gs-alpha-and-omega-classes-ignored-in-ie8).

**Comment by astratagem** [comment]
ie8 was ignoring class selectors because they were grouped with `:nth-child` pseudo-class using the SASS `@extend .alpha` – IE doesn't support `:nth-child`

fixed in 2460c7e048ee90c7072e337496b7ed471302282e by using the alpha/omega counter/alternator jawn
